tests/integration/test_script_runner.py

```diff
@@ -774,6 +775,7 @@ def _execute_integration_test(integration_test_fixture, tmp_path):
         loader.exec_module(test_script_module)
     except Exception as e:
         logger.error(str(e))
+        raise
```

Comment:
This is the line that was causing the issue

Reply:
❤️